packages/paste-core/core-bundle/tools/generate.js

```diff
@@ -31,7 +31,7 @@ function generateDependenciesFromPackageList(packageList) {
 function getAllCorePackages(packageList) {
   return packageList.filter(item => {
     const isCorePackage = item.location.includes('/paste-core/');
-    const isReleased = !item.private && item.version !== '0.0.1' && item.version !== '0.0.0';
+    const isReleased = !item.private;
```

Comment:
I kept noticing this, components not getting into core once you're merged them for the first time. Now that we have good private / public coverage and we do CD, we can relax this condition so they get added as you're building it. @richbachman